website/docs/configure_hydra/Intro.md

```diff
@@ -68,3 +68,17 @@ You can see the full Hydra config using `--cfg hydra`:
 - *hydra.runtime.version*: Hydra's version
 - *hydra.runtime.cwd*: Original working directory the app was executed from
 
+## Hydra default OmegaConf resolvers
+OmegaConf supports [interpolations](https://omegaconf.readthedocs.io/en/latest/usage.html#variable-interpolation) via [resolvers](https://omegaconf.readthedocs.io/en/latest/usage.html#custom-interpolations). Hydra supports the following resolvers by default.
+```yaml title="resolver.yaml"
+current_date: ${now:%Y-%m-%d}                              # call time.localtime() with pattern "%Y-%m-%d", eg: 2020-07-15
```

Comment:
I would describe each function individually. what is it, what parameter it takes.
In the example of now, there is no need for some many examples. just say it's using strftime (it's not localtime, look at the code carefully).